Systems/WorldGen/Standard/ChunkGen/1.GenTerra/GenTerra.cs

```diff
@@ -443,9 +436,11 @@ private void OnChunkColumnGen(IServerChunk[] chunks, int chunkX, int chunkZ, ITr
                                             int lZ = zN * lerpHor + z;
 
                                             double geoUpheavelTaper = 0;
-                                            if (posY > taperThreshold && disty[0] <- 2)
+                                            double distYHere = GameMath.BiLerp(distY[0], distY[1], distY[2], distY[3], x * lerpDeltaHor, z * lerpDeltaHor);
+                                            if (posY > taperThreshold && distYHere < -2)
                                             {
-                                                double upheavelness = -(GameMath.BiLerp(distortedposY0, distortedposY1, distortedposY2, distortedposY3, x * lerpDeltaHor, z * lerpDeltaHor) - posY) / geoUpheavalAmplitude;
+                                                double negativeClampedDistY = GameMath.Clamp(-distYHere, posY - mapsizeY, posY);

```

Comment:
Replaces clamps removed from lines 410, 412, 415, 417, but simplified to work on distY directly.